pkg/plugins/models.go

```diff
@@ -88,6 +88,7 @@ type PluginInclude struct {
 	DefaultNav bool            `json:"defaultNav"`
 	Slug       string          `json:"slug"`
 	Icon       string          `json:"icon"`
+	Uid        string          `json:"uid"`
 
 	Id string `json:"-"`
```

Comment:
can you add a function to this struct, like getUidOrSlug() then the logic in api/index.go could be simplifed

Reply:
Added as GetUidOrSlug().
I also removed the /dashboard/db/ link. Is that ok?

Reply:
Moved link logic to a new function in models.go